### ISSUE TITLE
Store selected set of persons and families in the clippings cart

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#177]
There is now in GVExport a unique function to select a set of persons and families. This is based on starting and stopping persons, choosing partners, siblings, relatives or anyone, and generation limits. This is comparable with the functionality of the old custom module "branch export" (see https://github.com/BlasiusSecundus/branch_export).

I like adding a function to copy all the selected persons and families and all related records (like media objects, notes, and sources) to the clippings cart. This would need a new button "copy to clippings cart" in the last line of the GVExport menu. Then you can use alternative visualization methods like TAM or LIN for the selected set of persons and families, too. And you can store the selected set in a GEDCOM file, which is not fully what is mentioned in issue #92, but a first step maybe.

What do you think about that? An alternative would be to copy the functionality of GVExport to select persons to the enhanced clippings cart module itself as a new function, but then it would be necessary to maintain two copies of this complex code. And because the selection of persons in GVExport is an interactive process of trial and error it is necessary to show intermediate results in a graphical user interface which is more than the enhanced clippings cart module can offer at the moment.

**Comment by Neriderc** [comment]
I think this could be considered an "output" of sorts. While you aren't downloading it, the module has completed it's part and is passing the output of the module to something else, in this case the clippings cart. So we could have this as an option in the Output settings.

After just a quick think I haven't come up with a good way of putting this in the GUI, but I do think this is a good idea. Possibly in an advanced section of the output settings.

I'm not familiar with this side of things and I know you are from your extended clippings cart module, would you be willing to add this functionality?

**Comment by hartenthaler** [comment]
Yes, I will provide a new function for this "output" to the clippings cart.

I would prefer that you design the way how to add a button to the GUI.

**Comment by Neriderc** [comment]
Thanks! I'm happy to do UI.

**Comment by Neriderc** [comment]
@hartenthaler are you actively working on this? If you are, I'll avoid touching any more code until you're done.

**Comment by hartenthaler** [comment]
Yes, I'm working on that now. I will inform you when I know more.
best regards
Hermann

**Comment by Neriderc** [comment]
Great, thanks for confirming. No rush, just wanted to check if you were working on it.

Cheers

**Comment by Neriderc** [comment]
How are you going with this? I've tried to avoid changes while you work on it, but there are a few things that need addressing. If you're almost done, I'll wait a little longer. If not, I'll proceed with other changes and we can handle merge conflicts if they arise.

**Comment by hartenthaler** [comment]
I'm sorry. Please go on. In the meantime, I'm a manager at the largest German genealogical community, Verein für Computergenealogie. That is more work than expected, but I'm still interested in using the power of the clippings cart .
best regards Hermann

**Comment by Neriderc** [comment]
Not a problem! Feel free to continue your work on this when you are able to, or I can take a look at some point in the future if you don't find yourself with the time.

**Comment by hartenthaler** [comment]
Now the name "GVExport" is more meaningful. You can show a part of your tree in GVExport and select interactive some persons/families, put them into the clippings cart and in the next step **export** them together with the related sources and media objects as a GEDCOM file. 
A great new feature! Thank you!

**Comment by Neriderc** [comment]
I've made this update now, to update the number of items in the clippings cart.
